doc/releases/release-notes-3.4.rst

```diff
@@ -300,20 +300,103 @@ Architectures
+  * Refactored and extended the scan delegator implementation, including
+    integration with broadcast sink.
+  * Added support for creating a broadcast sink from a PA sink.
+  * Added support for optional characteristics in CSIP.
+  * Implemented discovery by UUID instead of reading by UUID for multiple
+    characteristics.
+  * Added support for long reads and writes for multiple profiles.
+  * Added support for long BAP ASE notifications and optimized long notify
+    reads.
+  * Offloaded MCS notifications to the system workqueue.
+  * Added the CAP initiator cancel procedure.
+
 * Direction Finding
 
 * Host
 
+  * Updated the Host to the v5.4 specification.
+  * The GATT DB Hash is now recalculated upon loading settings.
+  * Added experimental support for SMP keypress notifications.
+  * Downgraded the severity of select log messages to avoid log flooding.
+  * Separated the handling of LE SC OOB pairing from the legacy OOB logic.
+  * Implemented the Encrypted Advertising Data feature.
+  * Added support for the new Periodic Advertising with Responses (PAwR), both
+    as an advertiser and as a scanner.
+  * Added support for initiating connections from PAwR, as well as receiving
+    connections while synced.
+  * Clarified the behavior that is enabled by the ``BT_PRIVACY`` Kconfig option.
+  * Introduced a new ``seg_recv`` L2CAP API for an applicaiton to receive
+    segmenets directly and manage credits explicitly.
+
 * Mesh
 
-  * Added experimental support for Mesh Protocol d1.1r18 specification.
+  * Added experimental support for Mesh Protocol d1.1r18 specification, gated
+    by a new configuration option. This includes:
+
+    * Enhanced Provisioning Authentication support.
+    * Mesh Remote Provisioning support including:
+      * Remote Provisioning Server and Client models.
+      * Composition Data Page 128 and Models Metadata Page 128 support.
+    * Large Composition Data support including:
+      * Large Composition Data Server and Client models.
+      * Models Metadata Page 0 support.
+    * New Transport Segmentation and Reassembly (SAR) implementation including:
+      * SAR Configuration Server and Client models.
+    * Mesh Private Beacons support including:
+      * Mesh Private Beacon Server and Client models.
+    * Opcodes Aggregator support including:
+      * Opcodes Aggregator Server and Client models.
+    * Proxy Solicitation support including:
+      * Solicitation PDU RPL Configuration Server and Client models.
+      * On-Demand Private Proxy Server and Client models.
+    * Composition Data Page 1 support.
+    * Other Mesh Profile Enhancements.
   * Added experimental support for Mesh Binary Large Object Transfer Model d1.0r04_PRr00 specification.
   * Added experimental support for Mesh Device Firmware Update Model d1.0r04_PRr00 specification.
+  * Fixed multiple profile errata: 11737, 11392, 11627, 11302, 11341
+  * Added async APIs for the Private Beacon Client and Large Comp Data Client.
```

Comment:
```
* Added async APIs for the Private Beacon Client and Large Comp Data Client.
* Added support for sending SNB and PRB in parallel.
```
Not a big deal, but there is no Zephyr version with Private Beacon but without parallel SNB and PRB etc.
Is it worth to separate these two lines from their parent functionality above?